CSharp/Samples/Microsoft.Bot.Builder.Abstractions.Teams/TeamsActivityProcessor.cs

```diff
@@ -260,5 +266,37 @@ private async Task<InvokeResponse> ProcessTeamsInvokeActivityAsync(ITurnContext
 
             return await this.invokeActivityHandler.HandleInvokeTaskAsync(turnContext).ConfigureAwait(false);
         }
+
+        private static string stripHtmlTag(string content)
+        {
+            var doc = new HtmlDocument();
+            doc.LoadHtml(content);
+            var TextRestrictedHtmlTags = new HashSet<string> { "at", "b", "i", "ss", "font", "u", "s", "pre" };
```

Comment:
I believe only `<at>` and `<attachment>` are enough. What's the other "hooks" being used in "body" that are not part of regular html? @RamjotSingh

Reply:
Then let's only keep the 2 elements for now, while stripe the others. (

Reply:
fixed